manifest.json

```diff
@@ -22,5 +22,16 @@
       "isRequired": true,
       "isBackendOnly": false
     }
+  },
+  "proxy": {
+    "whitelist": [
+      {
+        "url": ".*",
```

Comment:
Why is a completely open proxy required here?

Reply:
This was needed to get the favicons, just forgot to change

Reply:
Update works just fine
![image](https://user-images.githubusercontent.com/54243490/225864587-bf6e960b-4cbb-480e-aa9b-1dd95414aee6.png)